signac/linked_view.py

```diff
@@ -85,8 +79,29 @@ def create_linked_view(project, prefix=None, job_ids=None, path=None):
         for job in project.find_jobs():
             links["./job"] = job.path
         assert len(links) < 2
-    _check_directory_structure_validity(links.keys())
-    _update_view(prefix, links)
+
+    # Updating the view will fail on Windows, if symlinks are not enabled.
+    # Before re-raising the exception, print a helpful message for the expected error.
+    try:
+        _check_directory_structure_validity(links.keys())
+        _update_view(prefix, links)
+    except OSError as err:
+        if (sys.platform == "win32") & (err.winerror == 1314):
+            print("-----------------------------------------------------------------")
+            print(err.strerror)
+            print(" ")
+            print("You likely don't have permission to create Windows symlinks.")
+            print("To enable the creation of symlinks on Windows you need")
+            print("to enable 'Developer mode' (requires administrative rights).")
+            print(" ")
+            print("To enable 'Developer mode':")
+            print(" 1. Go to 'Settings'.")
+            print(" 2. In the search bar type 'Use developer features'")
+            print(" 3. Enable the item 'Developer mode'.")
+            print("The details for Home edition and between Windows versions may vary.")
+            print("-----------------------------------------------------------------")
+            print(" ")
```

Comment:
I'm going to rewrite this block with `textwrap.dedent` and a single string block.